src/ResizeSensor.js

```diff
@@ -163,19 +175,23 @@
             attachResizeEvent(element, callback);
         }
 
-        this.detach = function() {
+        this.detach = function(ev) {
             if (isCollectionTyped) {
                 var i = 0, j = element.length;
                 for (; i < j; i++) {
-                    ResizeSensor.detach(element[i]);
+                    ResizeSensor.detach(element[i], ev);
                 }
             } else {
-                ResizeSensor.detach(element);
+                ResizeSensor.detach(element, ev);
             }
         };
     };
 
-    ResizeSensor.detach = function(element) {
+    ResizeSensor.detach = function(element, ev) {
+        if(typeof ev == "function"){
+            element.resizedAttached.remove(ev);
+            if(element.resizedAttached.length()) return;
+        }
```

Comment:
Why is this necessary? The whole `element.resizedAttached` should be deleted, so I don't understand why you would want to remove only one event.